launcher/ui/dialogs/AboutDialog.cpp

```diff
@@ -97,12 +97,19 @@ QString getCreditsHtml()
     stream << "<br />\n";
 
     stream << "<h3>" << QObject::tr("With thanks to", "About Credits") << "</h3>\n";
+    stream << QString("<p>Davi Rafael %1</p>\n")    .arg(getWebsite("https://auti.one/"));
+    stream << QString("<p>Fulmine %1</p>\n")    .arg(getWebsite("https://www.fulmine.xyz/"));
+    stream << QString("<p>ely %1</p>\n")            .arg(getGitHub("elyrodso"));
+    stream << QString("<p>gon sawa %1</p>\n")       .arg(getGitHub("gonsawa"));
+    stream << QString("<p>Pankakes</p>\n");
+    stream << QString("<p>tobimori %1</p>\n")       .arg(getGitHub("tobimori"));
     stream << "<p>Orochimarufan &lt;<a href='mailto:orochimarufan.x3@gmail.com'>orochimarufan.x3@gmail.com</a>&gt;</p>\n";
     stream << "<p>TakSuyu &lt;<a href='mailto:taksuyu@gmail.com'>taksuyu@gmail.com</a>&gt;</p>\n";
     stream << "<p>Kilobyte &lt;<a href='mailto:stiepen22@gmx.de'>stiepen22@gmx.de</a>&gt;</p>\n";
     stream << "<p>Rootbear75 &lt;<a href='https://twitter.com/rootbear75'>@rootbear75</a>&gt;</p>\n";
     stream << "<p>Zeker Zhayard &lt;<a href='https://twitter.com/zeker_zhayard'>@Zeker_Zhayard</a>&gt;</p>\n";
-    stream << "<p>Everyone else who <a href='https://github.com/PolyMC/PolyMC/graphs/contributors'>contributed</a>!</p>\n";
+    stream << "<p>Everyone who helped establish our branding!</p>\n";
+    stream << "<p>And everyone else who <a href='https://github.com/PolyMC/PolyMC/graphs/contributors'>contributed</a>!</p>\n";
```

Comment:
```suggestion
    stream << "<p>And everyone else who <a href='https://github.com/PrismLauncher/PrismLauncher/graphs/contributors'>contributed</a>!</p>\n";
```
maybe update this already?